site/docs-md/cordova/migrating-from-cordova-to-capacitor.md

```diff
@@ -0,0 +1,100 @@
+## Migrating a Web App Using Cordova to Capacitor
+
+There are several steps required to fully migrate a web app using Cordova over to Capacitor. **Note:** It's recommended to work in a separate code branch when applying these changes.
+
+### Add Capacitor
+
+Begin by opening your project in a Terminal, then add Capacitor to [a web app](/docs/getting-started) or [an Ionic app](/docs/with-ionic). 
+
+Next, open `config.xml` and find the `id` field in the widget element. In this example, it's `io.ionic.myapp`.
+
+```xml
+<widget id="io.ionic.myapp" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+```
+
+Also find the `Name` of your app:
+
+```xml
+<name>MyApp</name>
+```
+
+Now, initialize Capacitor with this app information:
+
+```bash
+npx cap init [appName] [appId]
+```
+
+In this example, it would be `npx cap init MyApp io.ionic.myapp`. These values can be found in the newly created `capacitor.config.json` file.
+
+#### Build your Web App
+You must build your web project at least once before adding any native platforms.
+
+This ensures that the `www` folder that Capacitor has been [automatically configured](/docs/basics/configuring-your-app/) to use as the `webDir` in `capacitor.config.json` actually exists.
+
+#### Add Platforms
+
+Capacitor native platforms exist in their own top-level folders. Cordova's are located under `platforms/ios` or `platforms/android`.
+
+```bash
+npx cap add ios
+npx cap add android
+```
+
+Both android and ios folders at the root of the project are created. These are entirely separate native project artifacts that should be considered part of your app (i.e., check them into source control, edit them in their own IDEs, etc.). Additionally, any Cordova plugins already in `config.xml` are automatically installed by Capacitor into each new native project.
```

Comment:
This is not true, Capacitor doesn't read the config.xml.
But if the user did a `npm install`, as cordova adds the plugins as dependencies in package.json, capacitor will detect them, but it doesn't install them, they should be installed already.

Reply:
good catch thanks